main.go

```diff
@@ -1,11 +1,5 @@
 package main
 
-func Divine()  {
-	
-}
 func main() {
-i:=0
-i+=1
-println(i)
-println("done")
-}
+
+}
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>File is not `gofmt`-ed with `-s` (gofmt)